playbooks/openstack/README.md

```diff
@@ -183,14 +183,21 @@ Then run the provision + install playbook -- this will create the OpenStack
 resources:
 
 ```bash
-$ ansible-playbook --user openshift -i inventory \
+$ ansible-playbook --user openshift \
+  -i openshift-ansible/inventory/openstack/inventory.py
+  -i inventory \
```

Comment:
I didn't know multiple `-i` work!

Reply:
Yeah. We even mention it in the readme. It's pretty useful :-).